README.md

```diff
@@ -98,6 +98,10 @@ Generated documentation for the latest version of the Rust crates can be accesse
 
 To build, simply run `cargo build` (or `cargo build --release`).
 
+## What's new / Changelog
+
+See [CHANGELOG](./cedar-policy/CHANGELOG.md) and [Releases](https://github.com/cedar-policy/cedar/releases).
```

Comment:
Might be good to call out that this is the CHANGELOG for `cedar-policy`. The CLI has its own changelog I think